src/sentry/identity/bitbucket/provider.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+from sentry.pipeline import PipelineView
+from sentry.web.helpers import render_to_response
+from sentry.utils.http import absolute_uri
+from sentry.identity.base import Provider
+
+
+class BitbucketIdentityProvider(Provider):
+    key = 'bitbucket'
+    name = 'Bitbucket'
+
+    def get_pipeline_views(self):
+        return [
+            BitbucketLoginView(),
+        ]
+
+
+class BitbucketLoginView(PipelineView):
+    def dispatch(self, request, pipeline):
+        # TODO(LB): verify that user has access to this client key
+        client_id = request.GET.get('clientKey')
+
+        if client_id is None:
```

Comment:
style nit, can you swap the guard statements to check `is not None`, just since the `render_to_response` is longer, so it will fit better on screen if we reclaim those 4 spaces :)